lib/stats/series.js

```diff
@@ -0,0 +1,195 @@
+  })
+});
+
+export const workerUptime = new base.stats.Series({
+  name: 'worker_uptime',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time in MS that the system has been up
+    value: base.stats.types.Number
+  })
+});
+
+export const workerMemory = new base.stats.Series({
+  name: 'worker_memory',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // free/total in bytes
+    free: base.stats.types.Number,
+    total: base.stats.types.Number
+  })
+});
+
+export const workerHD = new base.stats.Series({
+  name: 'worker_hd',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // free/used/total in bytes
+    free: base.stats.types.Number,
+    used: base.stats.types.Number,
+    total: base.stats.types.Number
+  })
+});
+
+export const workerCPULoad = new base.stats.Series({
+  name: 'worker_cpu_load',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // cpu load expressed as a percentage
+    value: base.stats.types.Number
+  })
+});
+
+export const workerSpotTermination = new base.stats.Series({
+  name: 'worker_shutdown',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time node termination event emitted
+    value: base.stats.types.Number
+  })
+});
+
+export const volumeCacheHit = new base.stats.Series({
+  name: 'volume_cache_hit',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Cache name
+    value: base.stats.types.String
+  })
+});
+
+export const volumeCacheMiss = new base.stats.Series({
+  name: 'volume_cache_miss',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Cache name
+    value: base.stats.types.String
+  })
+});
+
+export const timeToFirstClaim = new base.stats.Series({
+  name: 'tasks_first_claim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    // Time in ms from when the task was created to when the first worker claimed
+    // it
+    value: base.stats.types.Number
+  })
+});
+
+export const claimTask = new base.stats.Series({
+  name: 'tasks_claim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const abortTask = new base.stats.Series({
+  name: 'tasks_abort',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const reclaimTask = new base.stats.Series({
+  name: 'tasks_reclaim',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const cancelTask = new base.stats.Series({
+  name: 'tasks_cancel',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const taskError = new base.stats.Series({
+  name: 'tasks_error',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
+  })
+});
+
+export const runTimeExceeded = new base.stats.Series({
+  name: 'tasks_run_time_exceeded',
+  columns: Object.assign({}, BASE_WORKER_SCHEMA, {
+    value: base.stats.types.Number
```

Comment:
How about we have one extra field here `maxRunTime` and `count: 1` which is always `1` because each observation is one...